aospy_synthetic/test/test_objs/calc_objs.py

```diff
@@ -0,0 +1,22 @@
+import projects
```

Comment:
Relative imports: `from . import models` etc.

And then I think its PEP8 to put relative imports after all other imports, so move `from aospy_synthetic...` above